02.calendar/cal.rb

```diff
@@ -0,0 +1,26 @@
+require 'optparse'
+require 'Date'
+
+options = ARGV.getopts('y:', 'm:')
+year  = options["y"].nil? ? Date.today.year  : options["y"].to_i
+month = options["m"].nil? ? Date.today.month : options["m"].to_i
+
+puts "      #{month}月 #{year}"
+puts "日 月 火 水 木 金 土"
+
+first_wday = Date.new(year, month, 1).wday
+print "   " * first_wday
+
+first_date = Date.new(year, month, 1)
+last_date = Date.new(year, month, -1)
+
+(first_date..last_date).each do |d|
```

Comment:
```suggestion
(first_date..last_date).each do |date|
```

`d`でも意味は伝わりますが、無理に略さなくてもいいのでは？と思いました。

[プログラミング初心者は変数名やメソッド名を略さない方がいいよ、という話 \- give IT a try](https://blog.jnito.com/entry/2020/10/20/092724)